src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1550,6 +1572,36 @@ function highlightRubricCol(index, questionNumber, highlight) {
 }
 
 /**
+ * Attaches event handlers to "assign weights" buttons to toggle the visibility
+ * of the input boxes for rubric weights
+ */
+function bindAssignWeightsButtons() {
+    $('body').on('click', 'a[id^="rubricAssignWeightsLink"]', function(e) {
```

Comment:
It is easier to maintain the code using this method instead of attaching event handlers directly to the buttons. See #3998